content/docs/guides/converting-wordpress-to-tina.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: Migrating WordPress Content
+last_edited: '2024-11-08T01:49:52.552Z'
+next: ''
+previous: ''
+---
+
+## Introduction
+
+This guide is a walkthrough using a **WordPress to Markdown converter**, which is recommended for most use cases where you want to migrate your content from WordPress to Tina.
+
+This guide focuses on transferring content and media only and will not include page styling.
+
+Other conversion options that are not covered in this guide include:
+
+- Using the WordPress Rest API - best for extracting the pure html for page or post content in JSON format
+- Using the WPGraphQL plugin - best for cases where you want to query and extract specific data and content
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/) is installed
+- You have set up a [Tina starter](https://tina.io/docs/introduction/using-starter)
+- Have a basic understanding of using the [Tina Editor](https://tina.io/docs/using-tina-editor) and [the schema](https://tina.io/docs/schema)
+
+## Getting Started
+
+### Converting WordPress to Markdown
+
+1\. Export your Wordpress content by going to [https://wordpress.com/export/](https://wordpress.com/export/) and select **Export All**
+
+> Name it **export.xml** to make it easier for you in the next step
+
+Use [wordpress-export-to-markdown](https://github.com/lonekorean/wordpress-export-to-markdown) to convert your WordPress content into Markdown.
+
+2\. In the folder where you've saved your exported file **export.xml**, run:
+
+```powershell
+npx wordpress-export-to-markdown --post-folders=false --include-other-types=true
+```
+
+3\. When prompted, make sure the filename is correct, and use the default values for the other prompts.
+
+This creates an `output` folder that groups your posts and pages into separate folders.
+
+### Import data to TinaCMS
+
+#### Migrating Images
+
+1\. In the output folder, drag the image folders `output/post/images` and `output/page/images` to your Tina app's `public/uploads` folder.
+
+> You can rename the folder before dragging to prevent duplicate image names being replaced
+
+> You can also set up alternative hosting with your preferred media manager.
+
+#### Migrating Posts
+
+1\. Navigate to your output's posts folder `output/post`, and run in the terminal:
+
+For MacOS/Linux:
+
+```powershell
+for file in *.txt; do mv "$file" "${file%.txt}.md"; done
+```
+
+For Windows:
+
+```powershell
+ren *.md *.mdx
+```
+
+Which will convert your files from Markdown to MDX.
+
+2\. Drag all the MDX files in `output/post` to your Tina app's `content/posts` folder, now your posts have been migrated to Tina!
+
+> This only works because the format of WordPress posts matches the format that the Tina Starter uses.
```

Comment:
```suggestion
> Note: This works because the format of WordPress posts matches the format that the Tina Starter uses.
```

Reply:
As per my conversation with @Marxoz, there shouldn't be "Note: " at the start to remain consistent with how the other pages on the docs are set out